content/slides/intro-to-jupyterlab.qmd

```diff
@@ -84,34 +115,56 @@ back into your desktop.
 :::
 
 
+## Desktop: logging out
+::: {.callout-warning}
+Do not use "Logout" from the desktop environment!
+:::
+
+:::{.notes}
+_TODO: add an image showing a "no" symbol over the logout menu? Or show what is
+supposed to be done? Just close the window? Something else?_
+:::
+
+
 ## Jupyter Notebooks
 
 :::::: {.columns}
-::: {.column width="50%"}
+::: {.column}
 * A tool for [Literate Programming](https://en.wikipedia.org/wiki/Literate_programming)
     * Showing your work
     * Tracking provenance
     * Telling a story
-:::
-
-::: {.column width="50%"}
+    
 * See also:
     * Quarto
     * JupyterBook
-    * _TODO: Others?_
+:::
+
+::: {.column}
+![Jupyter Notebook](/_media/jupyterlab_notebook.png)
 :::
 ::::::
 
 
 ## Jupyter Notebook example
 
-* Walk through running an example notebook and viewing its output in QGIS?
-    * _TODO: Finalize `demo.ipynb`. Include something simple for users to try changing._
+Live demo: Walk through running the `demo.ipnb` notebook.
 
+![Demo Notebook](/_media/jupyterlab_notebook.png)
```

Comment:
We have this image on two slides in a row. Maybe we could get rid of this slide and prompt to do a demo in the speaker notes of the previous slide?